deny.toml

```diff
@@ -129,4 +126,9 @@ skip = [
 
     # hashbrown uses an older version
     { name = "ahash", version = "=0.4.7" },
+
+    # rocksdb (transitively through clang-sys) uses this as a build
+    # dependency which conflicts with 0.6.7 that wasmer-engine-native
```

Comment:
Actually do not think that we need `wasmer-engine-native` at all.

Reply:
It’s used in `runtime/near-vm-runner/src/wasmer1_runner.rs`. It’s also a dependency on `wasmer` 1.0.2.

Reply:
OK, that's probably a mistake, but will fix it.

Reply:
cc @ailisp

Reply:
@olonho we provided configuration option for future switching to native engine here: https://github.com/near/nearcore/blob/f453895831ed423ec41628c034d5aae5e1164896/runtime/near-vm-runner/src/wasmer1_runner.rs#L319

We haven't tried wasmer2 has native engine + singlepass support yet, if it's supported that's still a better option than JIT engine. Note the wasmer 2 renaming: native engine -> DylibEngine; JIT engine -> UniversalEngine

Reply:
Yeah, we should remove support for dylib engine, filed https://github.com/near/nearcore/issues/4520 for that.